docs/java/features/odata/use-vdm.md

```diff
@@ -101,6 +101,25 @@ Request parameters:
    - `ignoreAnyVersionIdentifier()` will instead always send a `*` which acts as a wildcard to match all ETags.
 - All operations allow for adding custom headers via `withheader(...)`
 
+Navigable entities:
+- By using the `forEntity` method on service classes, you can recursively traverse navigation properties in entity sets. It's a generic API extension with type-safe methods to allow for requests along chained entities, according to their navigation properties.
```

Comment:
```suggestion
- By using the `forEntity` method on service classes you can directly access navigation properties in entity sets. It's a generic API extension with type-safe methods to allow for requests along chained entities, according to their navigation properties.
```

Reply:
Since this topic is a bit complicated I would try to word it a bit more, lets say, "direct"...

Reply:
Hmm.. Your proposal will hide the recursive nature of the API.

Reply:
True, then maybe add another sentence. Otherwise it's too unclear what we are talking about here IMO.

Reply:
I restructured the content. I'm no longer focusing on the recursive nature.